python/nav/web/api/v1/views.py

```diff
@@ -259,6 +289,10 @@ class NetboxViewSet(NAVAPIMixin, viewsets.ReadOnlyModelViewSet):
     filter_fields = ('ip', 'sysname', 'room', 'organization', 'category')
     search_fields = ('sysname', )
 
+    def pre_delete(self, obj):
+        obj.deleted_at = datetime.now()
+        obj.save()
```

Comment:
also, I don't understand how this is supposed to work. The API will set the timestamp, but still delete the actual Netbox entry right afterwards? Or does this override deletion, so that the background navclean process will remove the Netbox record, as is the case with deletion from SeedDB?

Reply:
Fixed without the use of pre_delete